VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
Will we be updating this file during deployments going forward? And this would reflect the service version?

Because certain changes will be external to this repo, it makes me wonder whether it's better to capture it externally to this repo. Maybe even on an endpoint that is served live and can report the underlying versions and details of a particular deployment.